.github/workflows/test.yml

```
@@ -106,3 +106,96 @@ jobs:
             Write-Error "File contents of downloaded artifacts are incorrect"
         }
       shell: pwsh
+
+  if-not-found-with-name:
+    name: Test `if-not-found` param wih name specified
```

```suggestion
    name: Test `if-not-found` param with name specified
```

(no one has seriously reviewed this PR)